src/utils/math.ts

```diff
@@ -0,0 +1,18 @@
+import { BackstopToken } from '@blend-capital/blend-sdk';
+
+/**
+ * Estimate the emissions apr for a reserve
+ * @param emissionsPerAsset emissions per asset
+ * @param backstopToken backstop token
+ * @param assetPrice asset price
+ */
+export function estimateEmissionsApr(
+  emissionsPerAsset: number,
+  backstopToken: BackstopToken,
+  assetPrice: number
+): number {
+  return (
+    (emissionsPerAsset * (backstopToken.lpTokenPrice / backstopToken.blndPerLpToken) * 0.8) /
```

Comment:
the blnd spot price calculated here is "technically" correct, but for clarity we might want to move to the official formula. It would also be good to add info about how `emissionsPerAsset` is also per year, and a float.

```js
const usdcPerBlnd = (FixedMath.toFloat(backstopToken. usdc, 7) / 0.2) / (FixedMath.toFloat(backstopToken.blnd, 7) / 0.8);
return (emissionsPerAssetPerYear * usdcPerBlnd) / assetPrice;
```